cmd/juju/controller/listmodels.go

```diff
@@ -232,6 +232,7 @@ type ModelSummary struct {
 
 	ControllerUUID     string                  `json:"controller-uuid" yaml:"controller-uuid"`
 	ControllerName     string                  `json:"controller-name" yaml:"controller-name"`
+	IsController       bool                    `json:"is-controller" yaml:"is-controller"`
```

Comment:
Also omitempty here too.

Reply:
This has the slightly awkward consequence of hiding the field from the user-visible output [when it's `false`](https://github.com/golang/go/issues/13284).

Reply:
After chatting with @wallyworld, we're going to remove `omitempty` from the structs within `cmd/...` so that the field is present when viewed in the cli